LuaMenu/widgets/chobby/components/battle/battle_list_window.lua

```diff
@@ -781,22 +787,13 @@ function BattleListWindow:OnUpdateBattleInfo(battleID)
 		imHaveMap:Invalidate()
 
 
-		--local imHaveGame = items.battleButton:GetChildByName("imHaveGame")
-		--imHaveGame.file = (VFS.HasArchive(battle.gameName) and IMG_READY or IMG_UNREADY)
-		--gameCaption:SetCaption(battle.gameName:gsub("_", " "))
+		local imHaveGame = items.battleButton:GetChildByName("imHaveGame")
+		if imHaveGame ~= nil then
+			imHaveGame.file = (VFS.HasArchive(battle.gameName) and IMG_READY or IMG_UNREADY)
+		end
 
 		local gameCaption = items.battleButton:GetChildByName("gameCaption")
-		local modeName = battle.battleMode and Configuration.battleTypeToHumanName[battle.battleMode]
```

Comment:
Displaying battle mode was removed with no configuration available.

Reply:
Displaying battle mode wasn't removed -> this functionality was moved to _MakeGameCaption as we were repeating the same code in two places.

The only functionality change was the addition of a imHaveGame checkbox which was disabled by default for ZK